### PR TITLE
thanks

### PR DESCRIPTION
**Pull request by AmitaiShmeeda**
hope this is good

### CODE REVIEW

File: path_to_folder.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    """
+    path = input('Enter path to folder: ')
+    list_of_files = os.listdir(path)
+    for file in list_of_files:
+        if file.startswith('deep'):
+            print(file)
+    return list_of_files
+
+
+# endregion
```
Comment (talbarfm) [inline path_to_folder.py]: Use _name==main_ - 
https://docs.python.org/3/library/__main__.html
https://www.geeksforgeeks.org/python-main-function/amp/

File: path_to_folder.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+import os
+
+
+def path_to_folder():
+    """function that receives a path to a folder,
+    and returns the list of all files that start with the sequence of letters "deep" in that folder.
+    """
+    path = input('Enter path to folder: ')
+    list_of_files = os.listdir(path)
+    for file in list_of_files:
```
Comment (talbarfm) [inline path_to_folder.py]: 1.  You should aim to write everything in a single line, even if you can. 
This link https://towardsdatascience.com/you-should-stop-craving-for-python-one-liners-5dbf8e0de23d explains it well, even though I disagree with how important memory and runtime are in most python programs. 

2.Seprate IO from logic - 
https://wiki.c2.com/?SeparateIoFromCalculation

File: path_to_folder.py (hunk 1 above)
Comment (AmitaiShmeeda) [reply path_to_folder.py]: ok!
how can i change the code to add this?

File: path_to_folder.py (hunk 2 above)
Comment (AmitaiShmeeda) [reply path_to_folder.py]: ok!
thanks, i want to change the code. what the best way to do it?

**Review by talbarfm — CHANGES_REQUESTED** [review]
.

**Pull request merged**